cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -404,7 +408,7 @@ private void deleteHandler(Id.Program programId, ProgramType type)
           stopProgramIfRunning(programId, type);
           break;
         case WORKFLOW:
-          scheduler.deleteSchedules(programId, ProgramType.WORKFLOW);
+          scheduler.deleteSchedules(programId, SchedulableProgramType.WORKFLOW);
```

Comment:
Do we only need to do this for workflows? If other program types can now be scheduled directly, don't we have to take care of their schedules when they are deleted?